### ISSUE TITLE
[BITV]: breadcrumbs home link without `href`

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#42624]
![Screenshot from 2024-01-08 14-08-34](https://github.com/nextcloud/server/assets/6078378/48c6ecf3-85a6-4e41-875e-16cb54992947)

`<a data-v-3ddab287="" title="Home" aria-label="Home">` have to have a valid `href` attr

**Comment by JuliaKirschenheuter** [comment]
@emoral435 could you please check if your PR is already in NcVue release? If not then it is not Done, but Parked

**Comment by emoral435** [comment]
Ah - I got mixed up on which PR this should go in! My bad, should be ready to be a11y28checked once https://github.com/nextcloud/server/pull/43064 gets merged!
